cocktail-info.html

```diff
@@ -0,0 +1,97 @@
+  <head>
+    <meta charset="utf-8">
+    <title>CocktailDB</title>
+    <link rel="stylesheet" href="styles/cocktail-info.css">
+    <link rel="stylesheet" href="styles/ingredients.css">
+    <script src="https://kit.fontawesome.com/483fec22dc.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header>
+      <div class="header-menu">
+        <div class="header-menu-title">
+          <h1><a><i class="fas fa-cocktail"></i>CocktailDB</a></h1>
+        </div>
+        <nav class="main-menu-navigation">
+          <ul class="menu-list" hidden>
+            <li id="catalog-link"><a href="index.html">Main Page</a></li>
+            <li id="create-link"><a href="create-cocktail.html">Create Cocktail</a></li>
+            <li id="signup-link"><a href="register.html">Sign up</a></li>
+            <li id="login-link"><a href="login.html">Log in</a></li>
+            <li id="logout-link"><a>Log out</a></li>
+          </ul>
+        </nav>
+      </div>
+      <form class="header-search" action="#" method="post">
+        <input type="text" name="search-input" placeholder="Search...">
+      </form>
+      <a class="header-login" href="login.html">Log in</a>
+      <a class="header-signup" href="register.html">Sign up</a>
+    </header>
+    <main>
+      <article class="cocktail-info">
+        <div class="cocktail-info-main">
+          <div class="cocktail-picture-info">
+            <div class="cocktail-image">
+                <img src="images/glass.png">
+                <div class="ingredient-container">
+                  <div class="cocktail-ingredient ingredient1 light-rum"></div>
+                  <div class="cocktail-ingredient ingredient2 water"></div>
+                  <div class="cocktail-ingredient ingredient3 lime-juice"></div>
+                </div>
+              </div>
+            <p class="cocktail-title">Mojito</p>
+            <div class="star-rating">
+              <input class="star-rating-input" id="star-rating-5" type="radio" name="rating" value="5">
+              <label class="star-rating-icon fa fa-star" for="star-rating-5"></label>
+              <input class="star-rating-input" id="star-rating-4" type="radio" name="rating" value="4">
+              <label class="star-rating-icon fa fa-star" for="star-rating-4"></label>
+              <input class="star-rating-input" id="star-rating-3" type="radio" name="rating" value="3">
+              <label class="star-rating-icon fa fa-star" for="star-rating-3"></label>
+              <input class="star-rating-input" id="star-rating-2" type="radio" name="rating" value="2">
+              <label class="star-rating-icon fa fa-star" for="star-rating-2"></label>
+              <input class="star-rating-input" id="star-rating-1" type="radio" name="rating" value="1">
+              <label class="star-rating-icon fa fa-star" for="star-rating-1"></label>
+            </div>
+          </div>
+          <div class="cocktail-details">
+            <ul class="ingredients">
```

Comment:
```suggestion
            <ol class="ingredients">
```

It is an ordered list.